src/services/change-requests.ts

```diff
@@ -17,13 +18,15 @@ import { useApiRequest } from './api-request';
 export const changeRequestTransformer = (changeRequest: ChangeRequest) => {
   const data: any = {
     ...changeRequest,
-    dateSubmitted: new Date(changeRequest.dateSubmitted),
+    dateSubmitted: new Date(changeRequest.dateSubmitted).toLocaleDateString(),
     dateReviewed: changeRequest.dateReviewed
-      ? new Date(changeRequest.dateReviewed)
+      ? new Date(changeRequest.dateReviewed).toLocaleDateString()
       : changeRequest.dateReviewed,
     dateImplemented: changeRequest.dateImplemented
-      ? new Date(changeRequest.dateImplemented)
-      : changeRequest.dateImplemented
+      ? new Date(changeRequest.dateImplemented).toLocaleDateString()
+      : changeRequest.dateImplemented,
+      submitterName: fullNamePipe(changeRequest.submitter),
+      wbsNum: wbsPipe(changeRequest.wbsNum)
```

Comment:
I don't believe this code should be changed. The purpose of this function is only to ensure that field which are supposed to be date objects are properly transformed to date objects. (There's an issue where they seem to stay as date strings, which is undesirable.)